smithy-rules-engine/src/main/java/software/amazon/smithy/rulesengine/language/syntax/parameters/ParameterType.java

```diff
@@ -51,7 +51,7 @@ public static ParameterType fromNode(StringNode node) throws RuleError {
         if (value.equalsIgnoreCase("boolean")) {
             return BOOLEAN;
         }
-        if (value.equals("stringArray")) {
+        if (value.equalsIgnoreCase("stringArray")) {
```

Comment:
I think we wanted to make it all `.equals()` but kept string/boolean as `equalsIgnoreCase` for backwards compatibility reason.